packages/Card/src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Avatar.styles";
+
+const propTypes = {
+  /** Avatar content. It can be initial as a string or icon */
+  children: PropTypes.node,
+  /** Background color of the Avatar */
+  color: PropTypes.oneOf(["black", "blue", "green", "orange", "pink", "red"]),
+  /** Randomize background color */
+  isRandomBackground: PropTypes.bool,
+  /** Size of Avatar */
+  size: PropTypes.oneOf(ShirtSizes.LIMITED),
+};
+
+const defaultProps = {
+  children: "C",
+  color: "black",
+  isRandomBackground: false,
+  size: "medium",
+};
+
+function Avatar(props) {
+  const { color, children, isRandomBackground, size } = props;
+
+  const avatarProps = {
+    color,
+    children,
+    isRandomBackground,
+    size,
+  };
+
+  if (isRandomBackground) {
+    const colors = ["black", "blue", "green", "orange", "pink", "red"];
+    const randomColor = colors[Math.floor(Math.random() * colors.length)];
+    avatarProps.color = randomColor;
+  }
+
+  return (
```

Comment:
we might need to accept the background and foreground for the text or icon. So dark and light contrasts can exist without a problem.

Reply:
@nahumzs the acl-ui Avatar component took in a string, and it decided on what colours to use, and what to display. That is not the plan anymore?

Reply:
Yes it is, but I think that shouldn't be part of the avatar component that should be part of another logic right? maybe like:
```js
import Avatar from "@paprika/avatar";

const colors = Avatar.getColors("my name");
<Avatar background={colors.background} color={colors.color} />
```

with this approach, you can either use it in a chat environment or in Assets and Attributes where in the future we might want to let the user set custom colours for their cards.

Reply:
In Assets, at the time when an Asset Type is created, a colour is assigned to it. Then that colour is always shown when the Asset Type is displayed. So we can't pick a random colour. (Although it totally makes sense if the component can do both)